faqs/galaxy/datatypes_fastq_format.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to format fastq data for tools that require .fastqsanger format
+area: datatypes
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+- Most tools that accept FASTQ data expect it to be in a specific FASTQ version: .fastqsanger. The .fastqsanger datatype must be assigned to each FASTQ dataset.
+
+Inorder to do that:
+
+- Watch the **[FASTQ Prep Illumina](http://vimeo.com/galaxyproject/fastqprep)** video for a complete walk-through.
+- Run **FastQC** first to assess the type.
+    - Run **FASTQ Groomer** if the data needs to have the quality scores rescaled.
+    - If you are certain that the quality scores are already scaled to Sanger Phred+33 (the result of an Illumina 1.8+ pipeline), the datatype ".fastqsanger" can be directly assinged. Click on the pencil icon {% icon galaxy-pencil %} to reach the **Edit Attributes** form. In the center panel, click on the "Datatype" tab, enter the datatype ".fastqsanger", and save. Metadata will assign, then the dataset can be used.
```

Comment:
```suggestion
    - If you are certain that the quality scores are already scaled to Sanger Phred+33 (the result of an Illumina 1.8+ pipeline), the datatype ".fastqsanger" can be directly assigned. Click on the pencil icon {% icon galaxy-pencil %} to reach the **Edit Attributes** form. In the center panel, click on the "Datatype" tab, enter the datatype ".fastqsanger", and save. Metadata will assign, then the dataset can be used.
```

Reply:
`Metadata will assign, then the dataset can be used.` <- can we rephrase this? or delete if not relevant?